docs/pages/introduction/faq.md

```diff
@@ -6,43 +6,103 @@ import { Collapsible } from '~/ui/components/Collapsible';
 
 In addition to the questions below, see the [forums](https://forums.expo.dev/) for more common questions and answers. Some of this information is repeated from earlier sections of the introduction but we include it here for comprehensiveness.
 
+<Collapsible summary="What is Expo used for?">
+
+Expo is an [open-source framework](https://github.com/expo/expo/) for React Native that allows you to develop cross-platform mobile applications using both JavaScript and React. Learn more about [what Expo offers](/introduction).
+
+</Collapsible>
+
+<Collapsible summary="Do companies use Expo?">
+
+Yes, Expo is used by top companies all around the world, serving hundreds of millions of end users. See our [showcase](https://expo.dev/customers) to learn more.
+
+</Collapsible>
+
+<Collapsible summary="Is Expo Go open source?">
+
+Yes, the source for Expo Go can be found in the [expo/expo GitHub repository](https://github.com/expo/expo) in the `home`, `ios`, and `android` directories. The Expo Go app is also built with Expo and React Native!
+
+</Collapsible>
+
+<Collapsible summary="Who created Expo?">
+
+Expo and Expo Application Services are developed by [650 industries](https://expo.dev/about). 650 industries a is a California-based company that was founded by [Charlie Cheever](https://en.wikipedia.org/wiki/Charlie_Cheever) and [James Ide](https://jameside.com/).
+
+</Collapsible>
+
+<Collapsible summary="Why does Expo have its own SDK?">
+
+When Expo was first created, React Native had yet to be publicly released, meaning there were no third-party packages. In order to make the developer experience of React Native reasonable, we created a number of libraries for common functionality. Many of these libraries have since been forked and modified to meet various needs, we welcome users to mix and match whichever custom native code they need to make their app great.
```

Comment:
```suggestion
When Expo was first created, React Native had yet to be publicly released. This means there were no third-party packages. To make the developer experience of React Native reasonable, we created several libraries to achieve common functionalities. Many of these libraries have since been forked and modified to meet various needs. We welcome users to mix and match whichever custom native code they need to make their app great.
```